### DISCUSSION TITLE
[QUESTION] Disable Default "Bold" and Default "Color"

### DISCUSSION THREAD

**Discussion by SakiSakiSakiSakiSaki** [Textualize/rich#3360] [Q&A]
Using a `Console()` object, how do I disable automatic **bold** when I use `console.print()`?

For example:
```py
console.print("Test(Test)")
```
Prints the following:
![image](https://github.com/Textualize/rich/assets/41088844/82f417d9-9719-4c85-8f66-166ed2b9cfcb)

For normal formatting, I have to do the below:
```py
console.print("[default not bold][not bold]Test(Test)[/default not bold]")
```
![image](https://github.com/Textualize/rich/assets/41088844/8b15519b-97ee-4cb7-a19b-2a68bb0836d2)

Defaulting everything to white may be overkill, so how would I disable it for just `"repr.call": Style(color="magenta", bold=True),` while specifying it within the Console object?

---
The solution may lie in, unsure:
```py
from rich.theme import Theme

console = Console(theme=Theme({})
```

**Comment by willmcgugan** [comment]
https://rich.readthedocs.io/en/latest/highlighting.html
